docs/UserGuide.md

```diff
@@ -191,7 +195,7 @@ Alternate Format: `delete INDEX…​`
 Examples:
 * `list` followed by `delete 2 5 7` deletes the 2nd, 5th and 7th person in the address book.
 
-### Removing specific tags from person: `removetag`
```

Comment:
A merge must have gone wrongly here, this is undoing changes I made to the user guide. 😢